devel/demangle/Portfile

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        eklitzke demangle master
+version             20160204
+
+categories          devel
+platforms           darwin
```

Comment:
You can eliminate this line, as it's now the default.

Reply:
@mascguy I've used this template: https://guide.macports.org/#development.examples

Maybe also clean it up?

Reply:
> @mascguy I've used this template: https://guide.macports.org/#development.examples
> 
> Maybe also clean it up?

Sounds good, I'll add that to our to-do list. Thanks for the heads-up!